### PR TITLE
Allow rendering and downsampling of stars proxy raster

### PR DESCRIPTION
**Pull request by lbusett**
Addresses #305

### CODE REVIEW

File: R/mapView.R [hunk 1]
```diff
@@ -881,65 +914,65 @@ setMethod('mapView', signature(x = 'sfc'),
 #'
 #' @describeIn mapView \code{\link{character}}
 setMethod('mapView', signature(x = 'character'),
```
Comment (tim-salabim) [inline R/mapView.R:916]: @lbusett why did you change things for the character method? Are you sure this is intended?

Comment (lbusett) [reply R/mapView.R:916]: Not at all - I only changed the "stars" method. I thought that the other highlighted changes were due to re-indentation, but maybe I changed something accidentally or while rebasing my local branch with remotes. 

Any change besides the stars method are definitely accidental... sorry - sometimes git has the better of me ;-) .

Comment (tim-salabim) [reply R/mapView.R:916]: No problem, but can you revert those changes?

Comment (lbusett) [reply R/mapView.R:916]: arg... I'll try... easiest way is to do a fresh clone...  ;-)

Comment (lbusett) [reply R/mapView.R:916]: can we delete this PR ? I'll try to do a clean one.

Comment (tim-salabim) [reply R/mapView.R:916]: sure, the cleaner the better, my PR skills are also limited

Comment (lbusett) [reply R/mapView.R:916]: ok. I tried a new one. Seems to be better.

**Comment by tim-salabim** [comment]
Closing as merged cleaner one
